jbmc/regression/jbmc-strings/append_obj_int_tostring/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE symex-driven-lazy-loading-expected-failure
+Main.class
+--lazy-methods --stop-on-fail --refine-strings --string-max-length 10 --unwind 10
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
+--
```

Comment:
Needs a comment explaining why this is a `symex-driven-lazy-loading-expected-failure`

Reply:
Yeah that sounds odd for this one?